yarn.lock

```diff
@@ -10307,7 +10307,7 @@ neo-async@^2.5.0, neo-async@^2.6.0, neo-async@^2.6.1:
   resolved "https://registry.yarnpkg.com/neo-async/-/neo-async-2.6.1.tgz#ac27ada66167fa8849a6addd837f6b189ad2081c"
   integrity sha512-iyam8fBuCUpWeKPGpaNMetEocMt364qkCsfL9JuhjXX6dRnguRVOfk2GZaDpPjcOKiiXCPINZC1GczQ7iTq3Zw==
 
-next-compose-plugins@2.2.0:
+next-compose-plugins@^2.2.0:
```

Comment:
@flybayer Were these previously pinned to a version instead of a range for a reason? If not then probably ok.

Reply:
Nice catch will pin them